### PR TITLE
Build script for MPSolve library by Leonardo Robol

### PR DESCRIPTION
**Pull request by kagalenko-m-b**
Build script for the [MPSolve library](https://github.com/robol/MPSolve) for finding the roots of polynomials and secular equations.

### CODE REVIEW

File: M/MPSolve/build_tarballs.jl [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+platforms = expand_cxxstring_abis(platforms)
+
+# The products that we will ensure are always built
+products = [
+    LibraryProduct("libmps", :libmps)
+]
+
+# Dependencies that must be installed before this package can be built
+dependencies = [
+    Dependency(PackageSpec(name="GMP_jll", uuid="781609d7-10c4-51f6-84f2-b8444358ff6d"))
```
Comment (giordano) [inline M/MPSolve/build_tarballs.jl:35]: We need to use an older version of gmp for the build (but I can't find it right now)

Comment (kagalenko-m-b) [reply M/MPSolve/build_tarballs.jl:35]: [This](one) ?

Comment (giordano) [reply M/MPSolve/build_tarballs.jl:35]: Yes, we need to build against v6.1.2:
```suggestion
    Dependency(PackageSpec(name="GMP_jll", uuid="781609d7-10c4-51f6-84f2-b8444358ff6d"), v"6.1.2"),
```

**Comment by giordano** [comment]
Thanks!

**Comment by kagalenko-m-b** [comment]
Appreciate your quick work to review this! Have nice day

**Pull request merged**